sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/rai_service.py

```diff
@@ -283,7 +299,7 @@ def _parse_content_harm_response(batch_response: List[Dict], metric_name: str) -
 
         # get content harm metric_value
         if "label" in harm_response:
-            metric_value = harm_response["label"]
+            metric_value = int(harm_response["label"])
```

Comment:
what if label is a non numeric string?

Reply:
Good question. I remember down the stack there was some assumption about this being a numeric value. Reverting this change alone will surface the issue when it is not int

Reply:
This was added by Ankit, and after a discussion, it's unclear if this is necessary. Removing for now, since all relevant tests appear to be passing without it.